.github/workflows/bot-testing-nitrogen.yml

```diff
@@ -20,7 +20,7 @@ concurrency:
 
 jobs:
   run-bot:
-    runs-on: [ledger-live-medium-linux]
+    runs-on: [ledger-live-4xlarge-linux]
```

Comment:
Is this still necessary now ?

Reply:
Not necessary but see description : 

`Moving to ledger-live-4xlarge-linux on nitrogen so we can test faster and runner availability looks better`

We use 'bot-testing-nitrogen' workflow for dev purpose, 'daily non-reg' can be slower so i did not update it

WDYT ? should we use the bigger runner everywhere ?